### ISSUE TITLE
inference issuse

### ISSUE THREAD

**Issue by deep-practice** [megvii-model/ShuffleNet-Series#32]
When inference using ShuffleNetV2,the class score is low and the class label is wrong.I use the following code:
transform = tansforms.Compose([
            OpencvResize(256),
            tansforms.CenterCrop(224),
            ToBGRTensor(),
        ])

model = ShuffleNetV2(model_size='2.0x')
model.load_state_dict(remove_prefix(torch.load("ShuffleNetV2.2.0x.pth.tar")["state_dict"],"module."))
model.eval()
image = Image.open("../Image/cat.jpg")
img = transform(image).unsqueeze(0)
output = model(img)
output=torch.softmax(output,dim=1)
vals,idxs=torch.max(output, dim=1)
print(vals[0].item(),idxs[0].item())

Output:
0.31983616948127747 287
287 means 'lynx, catamount'

Do I preprocess the input image incorecctly?

**Comment by nmaac** [comment]
Hi @deep-practice , 

please refer to the 'validate()' function in train.py (Line 241), there is an extra 'softmax' in your script.

**Comment by deep-practice** [comment]
I use softmax to get the probability of each class.It doesn't affect the results.
Is the data transform function right? @nmaac

**Comment by deep-practice** [comment]
And I just take the pipeline  of validate as a reference.

**Comment by nmaac** [comment]
The transform is the same with us. 
If your image "cat.jpg" is not from ImageNet, I suggest you test the Top-1 accuracy on the ImageNet validation set first, to ensure the correctness of your script.

**Comment by deep-practice** [comment]
I can get the correct result with mobilenet using the same picture.I just download your code and add a demo function.Nothing changed.So I'm confused what's wrong with my demo.

**Comment by nmaac** [comment]
Try removing ‘ToBGRTensor’, if your image is already in the BGR format.

**Comment by deep-practice** [comment]
I use PIL.Image module to read image,so I believe the input format is RGB.
